### PR TITLE
Bring kubernetes proof-of-concept setup with just collabora-online

### PR DESCRIPTION
**Pull request by yunusem**
Signed-off-by: Yunusemre Şentürk <yunusemre@collabora.com>
Change-Id: I5daeeea3746bb107b3c4e0b3b68efa0adacb773f

* Target version: master 

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: kubernetes/helm/README.md [hunk 1]
```diff
@@ -1,20 +1,50 @@
...
+example output :
+```
+|-----------|------------------|-------------|------------------------|
+| NAMESPACE |       NAME       | TARGET PORT |          URL           |
+|-----------|------------------|-------------|------------------------|
+| collabora | collabora-online |             | http://127.0.0.1:55064 |
+|-----------|------------------|-------------|------------------------|
+http://127.0.0.1:55064
+```
+so for this example http:// 127.0.0.1:55064 address should only response as "OK" on a plain html page, it means that it is ready to be used by nextcloud or owncloud of your local setup
```
Comment (timar) [inline kubernetes/helm/README.md:46]: it is ready to be used by your WOPI-like host of your local setup

File: kubernetes/helm/collabora-online/values.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Default values for libreoffice-online.
+# Default values for collabora-online.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 global:
   app:
-    name: libreoffice-online
-    version: 6.5.0
+    name: collabora-online
+    version: 6.4.4
```
Comment (timar) [inline kubernetes/helm/collabora-online/values.yaml:7]: Do we have to change version number here always, when we bump version of the release?

File: kubernetes/helm/collabora-online/values.yaml [hunk 3]
```diff
@@ -18,25 +18,26 @@ global:
...
+  tag: latest
 
 probes:
   urlScheme: HTTP # either HTTP or HTTPS
   port: 9980
   readiness:
     activate: true
     path: /
-    initialDelaySeconds: 180
+    initialDelaySeconds: 45
```
Comment (timar) [inline kubernetes/helm/collabora-online/values.yaml:40]: why 180 -> 45 ?

File: kubernetes/helm/collabora-online/values.yaml [hunk 4]
```diff
@@ -61,17 +62,17 @@ environmentVariablesLool:
   - name: extra_params
     value: --o:ssl.enable=false
   - name: username
-    value: 
+    value: peter
```
Comment (timar) [inline kubernetes/helm/collabora-online/values.yaml:65]: default user/password? I'd not use them.

**Review by timar — CHANGES_REQUESTED** [review]
See inline comments

File: kubernetes/helm/collabora-online/values.yaml (hunk 2 above)
Comment (yunusem) [reply kubernetes/helm/collabora-online/values.yaml:7]: I don't think so, I will test the setup without ^this version specification.

File: kubernetes/helm/collabora-online/values.yaml (hunk 3 above)
Comment (yunusem) [reply kubernetes/helm/collabora-online/values.yaml:40]: well I thought 3 minutes would be too much to wait just for loolwsd service to initialize

File: kubernetes/helm/collabora-online/values.yaml (hunk 4 above)
Comment (yunusem) [reply kubernetes/helm/collabora-online/values.yaml:65]: i will remove them

File: kubernetes/helm/collabora-online/values.yaml (hunk 3 above)
Comment (yunusem) [reply kubernetes/helm/collabora-online/values.yaml:40]: and in fact it was while testing

**Review by timar — APPROVED** [review]

File: kubernetes/helm/collabora-online/values.yaml [hunk 5]
```diff
@@ -18,17 +17,18 @@ global:
...
-      name: libreoffice-online
-      type: ClusterIP
+      name: collabora-online
+      type: NodePort # Use ClusterIP instead of NodePort if you are using standalone cluster structure 
       externalPort: 9980
       internalPort: 9980
   ingress: {}
 
-replicaCount: 3
+replicaCount: 3 # This is the amount of how many instances of loolwsd will run
```
Comment (fabianhick) [inline kubernetes/helm/collabora-online/values.yaml:26]: How do three replicas work properly? 
I thought the application is stateful when multiple users are editing the same file simultaneously?

**Pull request merged**